src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Sami32 
> And miracle that will make you a lier should come, isn't it ?

Sorry, I can't figure out what you mean by this. If you mean that I lied because I said that there was no support for ```_cuvid``` in the nightly I downloaded, I don't agree. There's so much I don't know about how to build FFmpeg that I don't try to guess what is actually supported or not, I simply test it and concluded that ```_cuvid``` is reported as ```Unknown decoder``` for H264. Because of that, I didn't understand why you'd think it would be valid for H265.

```powershell
Hardware acceleration methods:
d3d11va
dxva2
qsv
cuvid
```